usage/cli/src/main/java/brooklyn/cli/lister/ItemDescriptors.java

```diff
@@ -58,8 +65,17 @@
         List<Map<String, Object>> itemDescriptors = Lists.newArrayList();
         
         for (Class<? extends BrooklynObject> type : types) {
-            Map<String, Object> itemDescriptor = toItemDescriptor(type, headingsOnly);
-            itemDescriptors.add(itemDescriptor);
+            try {
+                Map<String, Object> itemDescriptor = toItemDescriptor(type, headingsOnly);
+                itemDescriptors.add(itemDescriptor);
+            } catch (Throwable throwable) {
+                if (throwable instanceof InterruptedException)
```

Comment:
you can get various `*Class*Error` instances here which we don't want to treat as fatal.
probably we should revisit `propagateIfFatal` not to crash and burn on arbitrary `Error` instances.